src/nativeresources/processrc.awk

```diff
@@ -54,12 +56,14 @@ BEGIN {
         gsub(/" +"/, "", $0);
         # remove all terminating newlines from the string - the msgfmt fails on those
         # since it expects them to be at the end of the msgid as well
-        while(gsub(/\\n"/, "\"", $0)) {}
+        if (removeTerm == 1) {
```

Comment:
Yep. If this is not an issue anymore, removing all these (parameter/this logic) would be cleaner. Will do.